src/pages/hotel/booking/booking-confirmation/payment-details/PaymentDetails.tsx

```diff
@@ -53,9 +53,10 @@ const PaymentDetails = ({ register, errors }: Props) => {
           className="rounded-sm bg-white card-details details border-color"
           type="text"
           id="cardHolderName"
+          data-testid="name"
```

Comment:
if possible avoid adding test id's to the elements. it is recommended to add only if you have no other options.
For input text elements, you can add aria-label/title instead and get that using query getByRole('textbox'...